dockstore-client/src/main/java/io/dockstore/client/cli/YamlVerifyUtility.java

```diff
@@ -71,10 +71,6 @@ private YamlVerifyUtility() {
     private static List<String> filesExist(List<String> paths, String base) {
         List<String> missingFiles = new ArrayList<>();
         for (String path : paths) {
-            if (path == null) {
```

Comment:
As noted in the Issue [description](https://github.com/dockstore/dockstore/issues/5443#issue-1642799968) this was a temporary fix to the issue of not having `DockstoreYamlHelper.readAsDockstoreYaml12` throw an error when a `.dockstore.yml` file is passed that **does not** contain a primary descriptor path or has an empty one.

Since `DockstoreYamlHelper.readAsDockstoreYaml12`  now catches this, this code is not required.